compat/test/browser/PureComponent.test.js

```diff
@@ -68,38 +68,44 @@ describe('PureComponent', () => {
 	});
 
 	it('should only re-render when props or state change', () => {
+		let set;
 		class C extends React.PureComponent {
+			constructor(props) {
+				super(props);
+				set = this.setState.bind(this);
+			}
+
 			render() {
 				return <div />;
 			}
 		}
 		let spy = sinon.spy(C.prototype, 'render');
 
-		let inst = React.render(<C />, scratch);
+		React.render(<C />, scratch);
 		expect(spy).to.have.been.calledOnce;
 		spy.resetHistory();
 
-		inst = React.render(<C />, scratch);
```

Comment:
Not entirely sure why calling `forceUpdate/setState` on this `Fragment` trickled through to the component that's confusing as hell